app/models/pipeline_run_stage.rb

```diff
@@ -168,37 +162,15 @@ def update_job_status
     if failed? || succeeded?
       unless job_log_id
         # set log id if not set
-        _job_status, self.job_log_id, _job_hash, self.job_description = job_info(job_id, id)
+        _job_status, self.job_log_id, self.job_description = job_info(job_id, id)
         save
       end
       return
     end
     # The job appears to be in progress.  Check to make sure it hasn't been killed in AWS.   But not too frequently.
```

Comment:
This comment makes not sense now, right?

Reply:
Actually it does, I haven't changed this logic. Maybe the method could have been originally phrased better, since it does more than just that.